tgui/packages/tgui/interfaces/IndividualStats/index.tsx

```diff
@@ -98,8 +98,8 @@ export const IndividualStats = (props) => {
     <Window
       theme={data.ui_theme}
       title={'Prep screen'}
-      width={850}
-      height={750}
+      width={980}
+      height={775}
```

Comment:
probably not an issue yet but be careful with UI sizes not everyone has a big monitor

Reply:
I don't think it should get any bigger than this now. Who the heck has anything less than 1280x720 tho?

Reply:
have you SEEN how zack plays